command.go

```diff
@@ -104,7 +104,18 @@ func (c *Command) Call(req *Request, re ResponseEmitter, env Environment) {
 		}
 	}
 
-	cmd.Run(req, re, env)
+	err = cmd.Run(req, re, env)
+	if err != nil {
+		if cmderr, ok := err.(cmdkit.Error); ok {
```

Comment:
IIRC, we also need to check if this is a pointer to a `cmdkit.Error`. Right?

Reply:
*sigh* you're right. We should figure something out to make this less of a hassle - see https://github.com/ipfs/go-ipfs-cmdkit/issues/19